src/SFA.DAS.LevyTransferMatching.Web/Controllers/PledgesController.cs

```diff
@@ -175,5 +176,21 @@ public async Task<IActionResult> Applications(ApplicationsRequest request)
 
             return View(response);
         }
+
+        [HttpGet]
+        [Route("{encodedPledgeId}/applications/{encodedApplicationId}")]
+        public async Task<IActionResult> Application(ApplicationRequest request,
+            CancellationToken cancellationToken = default)
+        {
+            var response = await _orchestrator.GetApplicationViewModel(request, cancellationToken);
+
+
+            if (response != null)
+            {
+                return View("~/Views/Pledges/ViewApplicantTransferPledge.cshtml", response);
```

Comment:
Could we just call this `Application.cshtml` and then we wouldn't have to explicitly specify the route/name as it'd be by convention?

Reply:
yeah changed